l4ping/l4ping.TESTGEN.ps1

```diff
@@ -0,0 +1,143 @@
+#===============================================
+# Script Input Parameters Enforcement
+#===============================================
+Param(
+    [parameter(Mandatory=$false)] [string] $Config = "Default",
+    [parameter(Mandatory=$true)]  [string] $DestIp,
```

Comment:
Parameter validation to make sure the string is a valid IP address.  Do we support both v4 and v6?

Reply:
I think l4ping only supports v4 based on some code that I found [here](https://microsoft.visualstudio.com/OS/_git/os.2020?path=/onecore/vm/dv/net/tools/l4ping/PingApp.cpp&version=GBofficial/main&line=30&lineEnd=45&lineStartColumn=1&lineEndColumn=2&lineStyle=plain&_a=contents).

Should I fix all the TESTGEN scripts? They all seem to be missing this validation. If I just add it here, I think it could break the PerfTest.ps1 script.

Reply:
Also, all the parameter validation is currently happening in validate_config to match how all the other TESTGEN scripts are doing it.